crowbar_framework/app/helpers/installer/upgrades_helper.rb

```diff
@@ -20,9 +20,21 @@ module Installer
   module UpgradesHelper
     def upgrade_continue_button(disabled = false)
       if disabled
-        button_tag icon_tag(:chevron_right, t(".continue")), class: "btn btn-primary disabled"
+        button_tag(
+          icon_tag(:chevron_right, t(".continue")),
+          class: "btn btn-primary disabled",
+          data: {
+            blockui: t(".blockui")
```

Comment:
isn't it better to be explicit about the i18n scope? which t(".blockui") is actually used here from which namespace?

as we have multiple `blockui: "message"` keys in different namespaces, I find it really confusing

same below